packages/twitch-pubsub-client/src/Messages/PubSubMessage.ts

```diff
@@ -4,6 +4,7 @@ import type { PubSubChatModActionMessage, PubSubChatModActionMessageData } from
 import type { PubSubRedemptionMessage, PubSubRedemptionMessageData } from './PubSubRedemptionMessage';
 import type { PubSubSubscriptionMessage, PubSubSubscriptionMessageData } from './PubSubSubscriptionMessage';
 import type { PubSubWhisperMessage, PubSubWhisperMessageData } from './PubSubWhisperMessage';
+import type { PubSubUndocumentedMessage, PubSubUndocumentedMessageData } from './PubSubUndocumentedMessage';
```

Comment:
I would prefer a terminology with "custom" or "arbitrary" (probably "custom" because it's easier to read and type) over "undocumented". Even though it's unlikely now that EventSub exists, there could be a new documented topic and I'm too slow to implement it, so then you could use this and not be confused by using a method named "undocumented" for documented things.